app/views/changeGroupLead/MainContactNameView.scala.html

```diff
@@ -36,15 +36,13 @@
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
-        @headingLabel(messages("mainContactName.heading", companyName))
+        @headingLabel(messages("mainContactName.heading", companyName), "value")
```

Comment:
"value" is defaulted for this component
```suggestion
        @headingLabel(messages("mainContactName.heading", companyName))
```